### ISSUE TITLE
When in dark mode the short keys dialog text is black instead of white or gray

### ISSUE THREAD

**Issue by prikeshsavla** [supabase/pg_graphql#291]
**Describe the bug**
When in dark mode the table contents are black

**To Reproduce**
1. Start graphiql on localhost:4000
2. Open settings and switch to dark mode
3. Open Short Keys dialog
4. View table

**Expected behavior**
The table contents should be light gray like rest of the text in Dark mode

**Screenshots**
![image](https://user-images.githubusercontent.com/34878926/208170259-e0aaa774-df55-465a-9815-413975e7ab0d.png)

**Versions:**
 - PostgreSQL: [e.g. 14.1]
 - pg_graphql commit ref: ea2ab60

**Additional context**
Should get fixed with the latest version of graphiql

**Comment by olirice** [comment]
The IDE from your screenshot is from https://github.com/graphql/graphiql so that would be the appropriate place to open this issue

pg_graphql is the GraphQL execution engine that GraphIQL communicates with. It doesn't have a UI component

**Comment by olirice** [comment]
sorry I may have misunderstood. Are you asking for the GraphIQL version to be updated in the pg_graphql docker-compose setup?

**Comment by prikeshsavla** [comment]
Yes, that is correct, I don't see that issue in the live demo of graphiql,
that's why I meant maybe updating it would work.


I can do that with a PR

**Comment by olirice** [comment]
Got it! a PR would be great, thanks 🙌

**Comment by olirice** [comment]
resolved by #292
